client/rest-high-level/src/test/java/org/elasticsearch/client/WatcherIT.java

```diff
@@ -72,4 +77,31 @@ public void testDeleteWatch() throws Exception {
         }
     }
 
+    public void testActivateWatch() throws Exception {
+        // Activate watch that exists
+        {
+            String watchId = randomAlphaOfLength(10);
+            createWatch(watchId);
+            ActivateWatchResponse activateWatchResponse1 = highLevelClient().watcher().activateWatch(new ActivateWatchRequest(watchId),
+                RequestOptions.DEFAULT);
+            assertThat(activateWatchResponse1.getStatus().state().isActive(), is(true));
+
+            ActivateWatchResponse activateWatchResponse2 = highLevelClient().watcher().activateWatch(new ActivateWatchRequest(watchId),
+                RequestOptions.DEFAULT);
+            assertThat(activateWatchResponse2.getStatus().state().isActive(), is(true));
+            assertThat(activateWatchResponse1.getStatus().state().getTimestamp(),
+                lessThan(activateWatchResponse2.getStatus().state().getTimestamp()));
+
+        }
+
+        // Activate watch that does not exist
```

Comment:
I'd split this into a separate test case, `testActivateWatchThatExists` and `testActivateWatchThatDoesNotExist`

Reply:
+1